src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocpackage/pkghtml/InputJavadocPackageHtmlIgnored.java

```diff
@@ -1,7 +1,10 @@
-package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocpackage.pkghtml;      // violation
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocpackage.pkghtml;
+/*
+allowLegacy = "false"
```

Comment:
move all config comments to most top
I do not think you need to use `"`

Reply:
Done.

Configs have been written properly and placed on top. 
All the unnecessary ```"``` have been removed from the configs.

Reply:
Also @romani refer to comment https://github.com/checkstyle/checkstyle/pull/12322#issuecomment-1327089006